From 8dab9f289a5e4aef4731b67fa14fa5d51172c704 Mon Sep 17 00:00:00 2001
From: Matt Jolly <kangie@gentoo.org>
Date: Sun, 21 Sep 2025 10:41:35 +1000
Subject: [PATCH] IWYU form_field_data.h

8ff2584bb5e7bceb0958561f97fcfd431f6fa6a6 migrated parsing to use
FormFieldData, and provided a forward decl. This is causing issues when
not building with C++ modules or PCH (i.e. downstream linux distro
build), so just include the header.

Signed-off-by: Matt Jolly <kangie@gentoo.org>
--- a/components/autofill/core/browser/form_parsing/autofill_scanner.h
+++ b/components/autofill/core/browser/form_parsing/autofill_scanner.h
@@ -11,9 +11,9 @@
 #include "base/containers/span.h"
 #include "base/memory/raw_span.h"
 
-namespace autofill {
+#include "components/autofill/core/common/form_field_data.h"
 
-class FormFieldData;
+namespace autofill {
 
 // A helper class for parsing a stream of |FormFieldData|'s with lookahead.
 class AutofillScanner {
-- 
2.50.1
